content/docs/config-files.md

```diff
@@ -119,6 +147,7 @@ Position | Name | Value format | Meaning
 3  |  Alt  |  "yes"/"no"  |  The key being mapped to has the Alt modifier
 4  |  Shift  |  "yes"/"no"  |  The key being mapped to has the Shift modifier
 5  |  Key  |  integer  |  The base virtual key number, in the 1..255 range
+6  | FolderName | string | Can be used to group one or more macros into a named sub-menu (new to v8.4.8)

```

Comment:
Is it worth stating that `FolderName` is optional?  And must be "hand-edited" in?